validate-charts/dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ghcr.io/openzipkin/alpine:3.12.1
+
+# Use latest recommended version here
+ARG HELM_VERSION=3.3.0
+
+WORKDIR /usr/local/bin
+
+# Install Helm, helm-gcs plugin, git and openssh-client
+RUN apk --update --no-cache add curl git openssh-client && \
```

Comment:
Given that this is no longer the main build image, it shouldn't need git or openssh.

Also nit: There's also no reason to clean up the image with the `del/rm` steps unless we're publishing it. Currently, it's just counterproductive since it's more work to do in each run (but maybe worth leaving since it's probably a very small amount of time and would be good if we ever decide to publish?).

Reply:
I would rather keep this in case we ever decide to publish. I have seen time it takes on test PR and it's in few seconds for complete execution so it's fine.

Reply:
That's fine - but just to clarify they were two separate comments - git + openssh are never needed any more (they were needed by circleci, since the image was also responsible for doing the checkout), so those should be removed. The second comment was about the cleanup which is what can stay in case we ever publish.

Reply:
Agree here that we should remove`git + openssh`.

Reply:
removing git is giving me error that git is required.

Reply:
ah - didn't notice that helm is using git for the plugin install. It probably doesn't need openssh if you're back in there since there's no auth happening, but who knows 🤷